tests/js-modules/modules.py

```diff
@@ -53,6 +53,26 @@ def test_module_import(network, args):
     return network
 
 
+@reqs.description("Test dynamic module import")
+def test_dynamic_module_import(network, args):
+    if args.package != "libjs_v8":
+        LOG.warning("Skipping test_dynamic_endpoints, requires V8")
+        return network
```

Comment:
You should be able to use the `@reqs.installed_package("libjs_v8")` decorator here, with the same effect.

Reply:
Done (both).

Reply:
Hm, I think this is not working. It ignores the condition if `--enforce-reqs` is not given, and if it's given then it fails the test.

Reply:
OK. Let me have a look at this. My first hunch is that we no longer need the `--enforce-reqs` argument anyway.

Reply:
Going to resolve this thread since it's not strictly related to this PR.